__tests__/integration/query.test.ts

```diff
@@ -241,14 +241,15 @@ describe("query", () => {
       if (e instanceof ServiceError) {
         expect(e.httpStatus).toBe(400);
         expect(e.code).toBe("constraint_failure");
-        expect(e.queryInfo?.summary).toBeDefined();
-        if (e.constraint_failures !== undefined) {
-          expect(e.constraint_failures.length).toBe(1);
-          for (let constraintFailure of e.constraint_failures) {
-            expect(constraintFailure.message).toBeDefined();
-            expect(constraintFailure.paths).toBeDefined();
-          }
-        }
+        expect(e.queryInfo).toMatchObject({
+          summary: expect.any(String),
+        });
+        expect(e.constraint_failures).toEqual([
+          {
+            message: expect.any(String),
+            paths: expect.arrayContaining([expect.any(Array)]),
```

Comment:
I couldn't figure out a simple way to assert the array contains `String | Number`. I guess we could extend `expect` but doesn't seem worth it. This is much improved already.